README.md

```diff
@@ -156,6 +156,7 @@
 - [ChungZH](https://github.com/ChungZH/ChungZH/)
 - [Orhun](https://github.com/orhun/orhun)
 - [Aveek Saha](https://github.com/Aveek-Saha/Aveek-Saha)
+- [CxyFreedom](https://github.com/cxyfreedom)
```

Comment:
Change it from `https://github.com/cxyfreedom` to `https://github.com/cxyfreedom/cxyfreedom`

Reply:
ok, I'll update that